locale/diglines.fr.tr

```diff
@@ -0,0 +1,23 @@
+# textdomain: digilines
+
+
+### inventory.lua ###
+
+Channel=Canal
+Digiline Chest=Coffre de digiline
+
+### lcd.lua ###
+
+Digiline LCD=Ecran à cristaux liquides digiline
+
+### lightsensor.lua ###
+
+Digiline Lightsensor=Capteur de lumière digiline
+
+### rtc.lua ###
+
+Digiline Real Time Clock (RTC)=Horloge de l'heure réelle digiline
```

Comment:
I would recommend `Horloge temps réel Digiline (RTC)` or `Horloge temps réel Digiline (HTR)`
(HTR is [french for RTC](https://fr.wikipedia.org/wiki/Horloge_temps_r%C3%A9el))

Reply:
I have done some modifications but I prefer to let "RTC"